src/components/ModalToggle.svelte

```diff
@@ -10,7 +10,7 @@
 <script lang="ts">
```

Comment:
I think `Popover.svelte` is a more appropriate name. (See also my comment above)

Reply:
bf9ca32d17ff6d78c80967f4a38843d0e380bebb